### PR TITLE
Create pyspark image

### PR DESCRIPTION
**Pull request by Sondergaard**
Create a pyspark image to be used by wholesale.
This version is trimmed and is about half the size of the one used before.

### CODE REVIEW

**Review by BjarkeMeier — DISMISSED** [review]
My approval is based on the anticipation that this equals the PR we left in geh-core.

**Comment by Sondergaard** [comment]
> My approval is based on the anticipation that this equals the PR we left in geh-core.

It is the same

File: .github/workflows/ci-orchestrator.yml
Comment (ShaBloze) [inline .github/workflows/ci-orchestrator.yml:1]: we should change the `create_release` job to ensure that a Release artifact is going to be published.

Comment (Sondergaard) [reply .github/workflows/ci-orchestrator.yml:1]: @ShaBloze - Please review the latest commit. Does that match your expectations?

Comment (ShaBloze) [reply .github/workflows/ci-orchestrator.yml:1]: Yes exactly.

**Review by ShaBloze — APPROVED** [review]
LGTM

**Review by djorgensendk — APPROVED** [review]

**Pull request merged**